docker-compose.yml

```diff
@@ -1,24 +1,49 @@
 version: '3'
 services:
-  web:
-    build: .
+
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: mysterium-api
     ports:
-      - "5000:5000"
+      - 8001:5000
+    environment:
+      DB_HOST: mysterium-mysql
+      DB_NAME: myst_api
+      DB_USER: myst_api
+      DB_PASSWORD: myst_api
+    volumes:
+      - .:/code
     depends_on:
       - db
 
   dashboard:
     build:
       context: .
       dockerfile: Dockerfile.dashboard
+    container_name: mysterium-dashboard
     ports:
-      - "8080:8080"
+      - 8002:8080
```

Comment:
Hmm, what this change gives?

Reply:
Docker inner ports doest not have to match to host ports

Reply:
It doesn't, but isn't it easier to remember 2 ports vs 4?

Reply:
Ok, made memorable ports:
- both applications runs on usual 80, because it's web services
- deployer may choose to map container there he wants (TEST, LIVE)
- LOCAL environment has 8001, 8002 - idea is that on localhost:800* - can serve all Mysterium microservices